help/en/html/hardware/cmri/CMRI.shtml

```diff
@@ -1264,29 +1273,39 @@ height="586"></p>
   <a name="documentation" id="documentation"></a>
   <h2 id="L737">Documentation</h2>
 
-  <h3 id="L739">JMRI Help</h3>
+  <h2 id="L739">JMRI Help</h3>
   <ul>
     <li><a href="Signals.shtml">C/MRI signals</a></li>
   </ul>
 
-  <h3 id="L746">Third Party info</h3>
+  <h2 id="L746">Third Party info</h3>
 
   <p>For more information on C/MRI (including the C/MRI User Manual), please
   see:</p>
   <ul>
     <li><a href="http://www.jlcenterprises.net/">JLC Enterprises web
     site</a></li>
     <li><a href="http://www.sliqelectronics.com/">SLIQ Electronics</a></li>
+  </ul>
+  <ul>
     <li><a href="https://groups.yahoo.com/neo/groups/CMRI_Users/info">C/MRI
       Users Yahoo group</a></li>
     <li><a href="http://www.modelrailroadcontrolsystems.com/">Model Railroad
       Control Systems (MRCS)</a></li>
   </ul>
 
+  <p></p>
```

Comment:
These empty paragraphs serve no purpose.

Reply:
Corrected lines in error.